tests/PhpSpreadsheetTests/Writer/Xlsx/Issue3443Test.php

```diff
@@ -78,6 +79,7 @@ public function testDefaultAlign(): void
         $rsheet = $reloadedSpreadsheet->getActiveSheet();
         $expected1 = [
             'horizontal' => 'general',
+            'justifyLastLine' => false,
```

Comment:
This will probably be affected by the change I suggested above to Reader/Xlsx/Styles.